CorePluginFramework/src/au/gov/asd/tac/constellation/pluginframework/logging/ConstellationLogger.java

```diff
@@ -34,47 +36,71 @@ public static ConstellationLogger getDefault() {
     }
 
     /**
-     * Called to indicate that Constellation is starting.
+     * Called to indicate that Constellation was started.
      */
-    public void applicationStart();
+    public void applicationStarted();
```

Comment:
I like the new names @cygnus-x-1 👍.
This is however an API change so can you please update CHANGELOG.md with the methods that have been renamed please?

Reply:
No worries - fixed.